dao/application_type_dao.go

```diff
@@ -51,8 +54,11 @@ func (a *ApplicationTypeDaoImpl) List(limit, offset int, filters []util.Filter)
 
 	// limiting + running the actual query.
 	result := query.Limit(limit).Offset(offset).Find(&appTypes)
+	if result.Error != nil {
+		return nil, 0, util.NewErrBadRequest(result.Error.Error())
+	}
```

Comment:
Same thing here

Reply:
This one is interesting, the first example (a table rename) _would_ in fact be a 500, but would most likely be caught in our integration tests luckily. So I wouldn't worry about catching that sort of error. I think in the near future when we flesh out the `/health` endpoint a bit more to hit the database it'll be really nice to have that response be something telling if the db can be hit, maybe even names of tables, etc. 

The other two though are indeed 400s. I would just leave them as 400s for now. We aren't validating the filters (just parsing them) in the middleware.

Reply:
so I can leave it as it is now? maybe I can do the change suggested by Mikel in the first comment
`+ err := query.Limit(limit).Offset(offset).Find(&applicationTypes).Error`

Reply:
That's up to you - I see the point but it's all the same to me.